toolkit/tools/scheduler/schedulerutils/buildworker.go

```diff
@@ -32,14 +32,15 @@ type BuildChannels struct {
 	Done             <-chan struct{}
 }
 
-// BuildRequest represents the results of a build agent trying to build a given node.
+// BuildRequest represents a work-order to a build agent asking it to build a given node.
 type BuildRequest struct {
-	Node           *pkggraph.PkgNode
-	PkgGraph       *pkggraph.PkgGraph
-	AncillaryNodes []*pkggraph.PkgNode
-	ExpectedFiles  []string
-	UseCache       bool
-	IsDelta        bool
+	Node           *pkggraph.PkgNode   // The main node being analyzed for the build.
+	PkgGraph       *pkggraph.PkgGraph  // The graph of all packages
+	AncillaryNodes []*pkggraph.PkgNode // For SRPM builds: other nodes stemming from the same SRPM. Empty otherwise.
+	ExpectedFiles  []string            // List of RPMs built by this node.
+	UseCache       bool                // Can we use a cached copy of this package instead of building it
+	IsDelta        bool                // Is this a pre-downloaded rpm (not traditional cache) that we may be able to skip rebuilding
+	Freshness      int                 // The freshness of the node (used to determine if we can skip building future nodes)
```

Comment:
Nit-picks out of lack of more serious things to worry about here:
```suggestion
	PkgGraph       *pkggraph.PkgGraph  // The graph of all packages.
	AncillaryNodes []*pkggraph.PkgNode // For SRPM builds: other nodes stemming from the same SRPM. Empty otherwise.
	ExpectedFiles  []string            // List of RPMs built by this node.
	UseCache       bool                // Can we use a cached copy of this package instead of building it.
	IsDelta        bool                // Is this a pre-downloaded RPM (not traditional cache) that we may be able to skip rebuilding.
	Freshness      int                 // The freshness of the node (used to determine if we can skip building future nodes).
```